status/status.go

```diff
@@ -29,88 +29,25 @@ package status
 
 import (
 	"context"
-	"errors"
 	"fmt"
 
-	"github.com/golang/protobuf/proto"
-	"github.com/golang/protobuf/ptypes"
 	spb "google.golang.org/genproto/googleapis/rpc/status"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/internal"
+	"google.golang.org/grpc/internal/status"
 )
 
+type Status = status.Status
```

Comment:
Please comment.  I believe the import symbol "status" is unknowable to readers of the godoc for this package, so we'll need a pointer to the package that now implements status.  I'm unsure whether we should use godoc or the package name, or both?

https://godoc.org/google.golang.org/grpc/internal/status
google.golang.org/grpc/internal/status

Reply:
Done. I added a little more to the comment, let me know what you think.

Reply:
LGTM thanks.